### ISSUE TITLE
[BUG] Single test fails

### ISSUE THREAD

**Issue by kevcenteno** [ReactTooltip/react-tooltip#1083]
**Bug description**
Test suites with only one test fail.

**Version of Package**
v5.21.1 but behaviour first seen in v5.20.0

**To Reproduce**
Delete all tests in `src/test/tooltip-props.spec.js` except for "basic tooltip".   Update snapshots to reduce noise.

OR

Replace `test` with `test.only` instead of deleting all the other tests.

Test run results in failure:
```
$ jest src/test/tooltip-props.spec.js
 FAIL  src/test/tooltip-props.spec.js
  tooltip props
    ✕ basic tooltip (1295 ms)

  ● tooltip props › basic tooltip

    Unable to find role="tooltip"

    Ignored nodes: comments, script, style
    <body>
      <div>
        <span
          id="basic-example"
        >
          Lorem Ipsum
        </span>
      </div>
    </body>

      24 |     let tooltip = null
      25 |
    > 26 |     await waitFor(() => {
         |                  ^
      27 |       tooltip = screen.getByRole('tooltip')
      28 |       expect(tooltip).toHaveAttribute('style')
      29 |     })

      at waitForWrapper (node_modules/@testing-library/dom/dist/wait-for.js:166:27)
      at Object.<anonymous> (src/test/tooltip-props.spec.js:26:18)
```

**Expected behavior**
Test should pass

**Desktop (please complete the following information if possible or delete this section):**
 - OS: ubuntu

**Comment by kevcenteno** [comment]
https://github.com/ReactTooltip/react-tooltip/commit/3b411f064ad36015bc93ef5095ef84d8040cb52e is the offending commit

**Comment by kevcenteno** [comment]
Any traction on this?

**Comment by gabrieljablonski** [comment]
@kevcenteno we're planning on overhauling the test suites soon, since there's been issues with inconsistency not just with this one.

Until then, if you think of any easy fixes for this one, we'd really appreciate suggestions or even a PR.

**Comment by gabrieljablonski** [comment]
Working on some test refactoring on #1171. Everything seems to be working now.

For future reference, when testing the tooltip **NOT** to be on screen, we recommend the following (using `@testing-library/react`):

```js
await waitFor(() => {
  expect(screen.queryByRole('tooltip')).not.toBeInTheDocument()
})
```

When getting a reference to the tooltip, we recommend this:

```js
const tooltip = await screen.findByRole('tooltip')
expect(tooltip).toHaveAttribute('style')
```

Waiting for the tooltip to have the `react-tooltip__show` (or `react-tooltip__closing`) class is often not necessary, but may be useful when testing both opening and closing the tooltip on the same spec.

```js
await waitFor(() => {
  // also useful with `react-tooltip__closing`
  expect(screen.queryByRole('tooltip')).toHaveClass('react-tooltip__show')
})
```

---

Will be closing this for now. Please open a new issue if you have further problems with writing tests for the tooltip. The internal tooltip tests should all pass after #1171 is merged.
